ledger-service/http-json-cli/src/main/scala/com/daml/http/SchemaHandling.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http
+import scalaz.std.option._
+import scalaz.syntax.traverse._
+
+private[http] sealed trait SchemaHandling
+private[http] object SchemaHandling {
+  private[http] case object CreateSchema extends SchemaHandling
+  private[http] case object Start extends SchemaHandling
+  private[http] case object CreateSchemaIfNeededAndStart extends SchemaHandling
+  private[http] case object CreateSchemaAndStart extends SchemaHandling
+
+  import scalaz.Validation.{success, failure}
+  import scalaz.Validation
+
+  private[http] def optionalSchemaHandlingField(
+      m: Map[String, String]
+  )(k: String): Either[String, Option[SchemaHandling]] = {
+    val parse: String => Validation[String, SchemaHandling] = {
```

Comment:
Looking around a bit more, we have a fairly similar option for Daml on SQL already https://github.com/digital-asset/daml/blob/c3a3d600832bf1926a2421302e31abc1219c8f5a/ledger/sandbox-common/src/main/scala/platform/sandbox/config/PostgresStartupMode.scala#L9.

The names don’t quite work but I think we can at least follow the format of using lower-case hyphenated names.

I think we can probably also drop the `Schema` given that it’s clear you’re talking about the db from the option name and then we could go from `create` to `create-only` although I don’t feel particularly strongly about the last one.

1. `create-only`
2. `start

Reply:
"Ist gebongt", consistency is good :)